internal/client/config/config.go

```diff
@@ -90,69 +43,30 @@ func New(c *cli.Context) (Config, error) {
 }
 
 func (c *Config) Init() (err error) {
-	c.Severities = c.splitSeverity(c.severities)
-	c.VulnType = strings.Split(c.vulnType, ",")
-	c.AppVersion = c.context.App.Version
-	c.CustomHeaders = splitCustomHeaders(c.customHeaders)
-
-	// add token to custom headers
-	if c.token != "" {
-		c.CustomHeaders.Set(c.tokenHeader, c.token)
-	}
-
 	// --clear-cache doesn't conduct the scan
 	if c.ClearCache {
 		return nil
 	}
 
-	ctx := c.context
-	if c.Input == "" && ctx.NArg() == 0 {
-		c.logger.Error(`trivy requires at least 1 argument or --input option`)
-		cli.ShowAppHelp(c.context)
-		return xerrors.New("arguments error")
-	} else if ctx.NArg() > 1 {
-		c.logger.Error(`multiple images cannot be specified`)
-		return xerrors.New("arguments error")
-	}
+	c.CustomHeaders = splitCustomHeaders(c.customHeaders)
```

Comment:
I'm trying to understand how does this work. How can I pass custom headers? Does this make it a flag like `--custom-headers`? https://github.com/aquasecurity/trivy/blob/507def6de7e77afbba2330c69153cfd4aa71d12c/internal/client/config/config.go#L41

Reply:
True.